src/Core/Math/Ellipsoid.js

```diff
@@ -56,10 +56,10 @@ Ellipsoid.prototype.setSize = function setSize(size) {
 };
 
 
-Ellipsoid.prototype.cartographicToCartesian = function cartographicToCartesian(coordCarto) {
+Ellipsoid.prototype.cartographicToCartesian = function cartographicToCartesian(coordCarto, normal) {
```

Comment:
You must change the code to not modify the `normal` given by the user (modifying parameters makes the API error-prone)

Reply:
done